lblod/app-poc-form-builder
# SFB-100: create domain for concept & concept-scheme + migrations for the schemes

## ID
 [SFB-100](https://binnenland.atlassian.net/browse/SFB-100)

 ## Description

The concept schemes are fetched with a direct sparql query in the frontend. To make this possible through ember-data we need to create a domain in the backend so this is possible. Next to creating the domains we are adding all the concept-schemes with their concepts to the application graph so they can be fetched. by the ember-data models.
 ## Type of change


 ## How to test

Check the possible concept-schemes that are available right now in the form builder application (for example check it with codelijsten on [dev](https://dev.form-builder.lblod.info/codelijsten)) against the concept-schemes that are available in this PR.

 ## Links to other PR's

 - https://github.com/lblod/frontend-form-builder/pull/66
